### PR TITLE
Add jobs option to ctest

### PR DESCRIPTION
**Pull request by jeffin143**
Let's see if this makes any difference

### CODE REVIEW

File: .ci/linux-steps.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ steps:
   displayName: 'Build'
 
 # Run tests via ctest.
-- script: cd build && CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test .
+- script: cd build && CTEST_OUTPUT_ON_FAILURE=1 ctest -j8 -T Test .
```
Comment (zoq) [inline .ci/linux-steps.yaml:73]: Azure pipelines runs on `Standard_DS2_v2` machines (https://docs.microsoft.com/en-us/azure/virtual-machines/dv2-dsv2-series#dsv2-series) which has two virtual cores, so not sure it makes sense to use more than two here.

**Comment by shrit** [comment]
The issue/pull request number #3000 :+1:

File: .ci/linux-steps.yaml (hunk 1 above)
Comment (shrit) [reply .ci/linux-steps.yaml:73]: Agree with @zoq the `-j8`will work, but it will overload the VM, and we will not gain any time.

**Comment by rcurtin** [comment]
Just judging by other PRs, doesn't look like a serious speed boost or anything.  Maybe `-j2` will help?  If not, it could be because the tests themselves are already using all the cores that are available.

**Pull request closed without merging**